src/pbcontroller.cpp

```diff
@@ -299,8 +298,8 @@ int PbController::run()
 
 	std::cout << _("done.") << std::endl;
 
-	ql = new QueueLoader(queue_file, cfg,
-		std::bind(&PbController::set_view_update_necessary, this, true));
+	ql.reset(new QueueLoader(queue_file, cfg,
```

Comment:
Looking at podboat.cpp, it's obvious that `run()` runs for almost the entire lifetime of `PbController`, and the object is not re-used afterwards. Thus, is it possible to make `QueueLoader` an ordinary, non-pointer member of `PbController`?

Reply:
It looks like that requires some more formatting which I rather do at a later time (in a different PR).
The refactoring would be needed to make the value for `queue_file` available while initializing `ql` in the `PbController` initializer list.

Reply:
Ah, right, because `queue_file` has to be initialized with XDG/dotfile logic. Got it, thanks for the explanation!